zerver/tests/test_management_commands.py

```diff
@@ -112,4 +112,4 @@ def test_realm_creation_with_expired_link(self):
 
             result = self.client.get(generated_link)
             self.assertEquals(result.status_code, 200)
-            self.assertIn("The organization creation link has been expired or is not valid.", result.content)
+            self.assertIn("The organization creation link has been expired or is not valid.", result.content.decode('utf-8'))
```

Comment:
That's because I added `assert_in_response` to `AuthedTestCase` and this test class doesn't inherit from `AuthedTestCase`.

I can make another class which implements `assert_in_response` and make all required test classes inherit from it if you think that's a good approach. What should I name that class? Is `ZulipTestCase` okay?